Makefile

```diff
@@ -1,42 +1,54 @@
+MAKEFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
```

Comment:
Is this used?

Reply:
No. See https://github.com/Vorne/pru-driver/pull/11/commits/e793535916150e17835923fe343ee31de67969d5.